_sass/rootvariables.scss

```
@@ -1,118 +1,119 @@
-    --action-color-dark: #EBC017;
-    --action-color-dark-hover: #C09E0F;
-    --action-color-light: #FFF3C5;
-
-    --color-summary-background:#333;
-    --color-summary-background-hover:#666666;
-    --color-details-background:#282828;
-    --color-components-background: #202020;
-}
+:root{
+
+    // Colors that will not change in the dark mode.
```

Stylelint (17.14.0)

[error] Unexpected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)